openslides/users/migrations/0002_auto_20150630_0143.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
No we can't. Try to remove this file and run OpenSlides with django 1.8.

If a database was create with the alpha the User and Group database was created with the first migration file. So the database tables have to be changed.